src/components/Input/styledComponents.tsx

```diff
@@ -0,0 +1,118 @@
+import styled, { css } from "styled-components";
+
+import { SuggestionProps } from "./interface";
+
```

Comment:
```suggestion
```

Reply:
Unless we're adding a new line before assets import.

Reply:
This was my way of thinking to keep those things seperated.